ui/partials/perf/graphsctrl.html

```diff
@@ -13,11 +13,15 @@
             </button>
             <table>
               <tr>
-                <td width="10px" class="graph-legend" style="padding: 4px; background-color: {{series.color}}"></td>
+                <td width="10px" class="graph-legend" style="padding: 4px; background-color: {{series.mycolor}}"></td>
                 <td class="graph-legend">
                   <b>{{series.name}}</b><br/>
                   {{series.projectName}}<br/>
                   {{series.platform}}<br/>
+                  <div ng-show="series.inactive">
```

Comment:
It isn't necessary to display this explicitly, we can just style the series appropriately if it's inactive.